jenkins/scripts/bare_metal_lab/deploy-lab.yaml

```
@@ -2,14 +2,16 @@
   vars:
     bml_ilo_username: "{{ lookup('env', 'BML_ILO_USERNAME') }}"
     bml_ilo_password: "{{ lookup('env', 'BML_ILO_PASSWORD') }}"
+    github_token: "{{ lookup('env', 'GITHUB_TOKEN') }}"
   environment:
     EPHEMERAL_CLUSTER: "minikube"
     EXTERNAL_VLAN_ID: 3
+    GITHUB_TOKEN: "{{ lookup('env', 'GITHUB_TOKEN') }}"
   tasks:
     - name: Check required env vars are set
       fail:
-        msg: Ensure that BML_ILO_USERNAME and BML_ILO_PASSWORD environment variables are set
-      when: not bml_ilo_username or not bml_ilo_password
+        msg: Ensure that BML_ILO_USERNAME, BML_ILO_PASSWORD and GITHUB_TOKEN environment variables are set
+      when: not bml_ilo_username or not bml_ilo_password or not github_token
```

it is weird why CI is failing here, can we perhaps try with 

```suggestion
        msg: Ensure that BML_ILO_USERNAME, BML_ILO_PASSWORD and GITHUB_TOKEN environment variables are set
      when: not (bml_ilo_username or bml_ilo_password or github_token)
```

---

disregard, this didn't help

---

I wonder if the variable is not getting exported properly from Jenkins